src/test/js/App.test.js

```diff
@@ -53,24 +59,42 @@ describe('Test adding questions', () => {
         wrapper.find(Select).find({ labelId: "qtype_select_label" }).at(0).simulate('change', { target: { value: "numberQuestion" } } );
         expect(wrapper.find({ label: "Open Question Input" }).length).toBe(0);
         expect(wrapper.find({ label: "Number Question Input" }).length).toBe(0);
+        expect(wrapper.find({ label: "Dropdown Question Input" }).length).toBe(0);
 
         wrapper.find(Button).find({ label: "Add Question" }).at(0).simulate('click');
         expect(wrapper.find({ label: "Open Question Input" }).length).toBe(0);
         expect(wrapper.find({ label: "Number Question Input" }).length).toBe(1);
+        expect(wrapper.find({ label: "Dropdown Question Input" }).length).toBe(0);
+   });
+
+   it('Add a single dropdown questions', () => {
+        let wrapper = shallow(<App />);
+
+        wrapper.find(Select).find({ labelId: "qtype_select_label" }).at(0).simulate('change', { target: { value: "dropdown" } } );
+        expect(wrapper.find({ label: "Open Question Input" }).length).toBe(0);
+        expect(wrapper.find({ label: "Number Question Input" }).length).toBe(0);
+        expect(wrapper.find({ label: "Dropdown Question Input" }).length).toBe(0);
+
+        wrapper.find(Button).find({ label: "Add Question" }).at(0).simulate('click');
+        expect(wrapper.find({ label: "Open Question Input" }).length).toBe(0);
+        expect(wrapper.find({ label: "Number Question Input" }).length).toBe(0);
+        expect(wrapper.find({ label: "Dropdown Question Input" }).length).toBe(1);
    });
 
    it('Add a question', () => {
 
         let wrapper = shallow(<App />);
         let sumOpen = 0;
         let sumNumber = 0;
+        let sumDrop = 0
         let currentType = '';
 
         for (let i = 0; i < 10; i++) {
-            // We test two things, do we
-            var toChange = Math.random() < 0.5;
+            // We test two things, do we change the value, and if so what is the new value
+            let toChange = Math.random() < 0.5;
```

Comment:
Not a necessary change, but just as a heads up, you can use `const` instead of let anytime that variable is not re-declared farther down the code. Even though it's in a loop, since it redeclares it as a new variable everytime, you're never mutating it so `const` is fine here.